examples/src/main/java/io/druid/examples/ExamplesDruidModule.java

```diff
@@ -38,9 +39,8 @@
   {
     return Arrays.<Module>asList(
         new SimpleModule("ExamplesModule")
-            .registerSubtypes(
-                new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer")
-            )
+            .registerSubtypes(new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer"))
+            .registerSubtypes(new NamedType(GuavaDoubleMath.class, "guavaDoubleMath"))
```

Comment:
Would it be natural to be in example module for examples? GuavaDoubleMath is good but I don't think anyone needs that as an extension.